docs/General/createProfileACM.md

```diff
@@ -61,10 +65,10 @@ In this example, the hostname is **cb-vending1** and the DNS suffix is **burgerb
 
 4. Under **Activation**, select **Admin Control Mode** from the dropdown menu.
 
-5. Leave **Generate Random AMT Password** unchecked.
+5. In order to facilitate quick set up,  leave **Generate Random AMT Password** unchecked. 
    
     !!! tip "Production Environment"
-        In a production environment, you typically generate a random password for each AMT device to create a stronger, more secure AMT environment.
+        In a production environment, generate a random password for each AMT device to create a stronger, more secure AMT environment. However, if opting to use random passwords, be aware of the risks. **If the SQL database is lost or corrupted, all randomly generated credentials will be lost.** There will be no way to login. The administrator will have to clear the CMOS battery on the managed devices!
```

Comment:
This should be if the Vault database is lost - Vault holds the passwords and is the more important portion. SQL could be lost, and be restored if the vault was never lost and you'll still be good.

Reply:
Oh, good! Great catch. I was going off Corey's explanation, and I obviously missed the part about Vault. Excellent.